src/job.py

```diff
@@ -60,6 +60,7 @@ class Job:
         destination (Destination[Any]): The destination to load data into
     """
 
+    name: str
```

Comment:
This is great. Maybe we could include a log somewhere that references it. Or did we already have one but it just looked weird?

Reply:
Yes we did, that's exactly why I added the `name` attribute.

Try a normal run of the `main`. 
Before:
```
/home/az/.local/share/virtualenvs/dune-sync-_TGGYsHG/bin/python /home/az/workspace/dune-sync/src/main.py 
2024-11-23 11:14:57,721 INFO dune_client.api.base executing 4238114 on medium cluster
2024-11-23 11:14:57,728 INFO dune_client.api.base executing 4273244 on medium cluster
2024-11-23 11:14:58,122 INFO dune-sync Job completed: Job(source=<src.sources.dune.DuneSource object at 0x7f97d075e5d0>, destination=<src.destinations.postgres.PostgresDestination object at 0x7f97d85fbb30>)
2024-11-23 11:14:58,361 INFO dune_client.api.base waiting for query execution 01JDC88ZETRRWQ981FKPGSQD4M to complete: ExecutionState.PENDING (queue position: 1)
2024-11-23 11:14:58,368 INFO dune_client.api.base waiting for query execution 01JDC88ZF0QM34MPHPG7GBDE68 to complete: ExecutionState.PENDING (queue position: 2)
2024-11-23 11:15:03,531 WARNING dune-sync Unknown column: row(appcode varchar, appdataparams row(appcode varchar, environment varchar), backend row(hooks row(post array(row(calldata varchar, gaslimit varchar, target varchar)), pre array(row(calldata varchar, gaslimit varchar, target varchar)))), environment varchar, fullappdata varchar, metadata row(environment varchar, hooks row(post array(row(calldata varchar, gaslimit varchar, target varchar)), pre array(row(calldata varchar, gaslimit varchar, target varchar)), version varchar), orderclass row(orderclass varchar, version varchar), partnerfee row(bps bigint, recipient varchar), quote row(buyamount varchar, sellamount varchar, slippagebips varchar, version varchar), referrer row(address varchar, kind varchar, referrer varchar, version varchar), utm row(utmcampaign varchar, utmcontent varchar, utmmedium varchar, utmsource varchar, utmterm varchar, version varchar), widget row(appcode varchar, environment varchar)), metadataparams row(orderclassparams row(orderclass varchar), quoteparams row(slippagebips varchar), utmparams row(utmsource varchar)), version varchar) - treating as JSONB
2024-11-23 11:15:03,589 INFO dune-sync Data saved to app_data successfully!
2024-11-23 11:15:03,589 INFO dune-sync Job completed: Job(source=<src.sources.postgres.PostgresSource object at 0x7f97d075eae0>, destination=<src.destinations.dune.DuneDestination object at 0x7f97d07d0bc0>)
2024-11-23 11:15:03,591 WARNING dune-sync Unknown column: array(row(address varbinary, storagekeys array(varbinary))) - treating as JSONB
2024-11-23 11:15:03,637 INFO dune-sync Data saved to results_4238114 successfully!
2024-11-23 11:15:03,637 INFO dune-sync Job completed: Job(source=<src.sources.dune.DuneSource object at 0x7f97d07ec410>, destination=<src.destinations.postgres.PostgresDestination object at 0x7f97d07ec650>)
```

With this change:
```
2024-11-23 11:14:06,862 INFO dune_client.api.base executing 4238114 on medium cluster
2024-11-23 11:14:06,869 INFO dune_client.api.base executing 4273244 on medium cluster
2024-11-23 11:14:07,235 INFO dune-sync Job completed: Sync Parameterized Dune Query with Multiple Types to Postgres
2024-11-23 11:14:07,492 INFO dune_client.api.base waiting for query execution 01JDC87DRRTGABY8JARK8TXDST to complete: ExecutionState.PENDING (queue position: 1)
2024-11-23 11:14:07,499 INFO dune_client.api.base waiting for query execution 01JDC87DRS795V54WN1GCP2BFR to complete: ExecutionState.PENDING (queue position: 2)
2024-11-23 11:14:12,676 WARNING dune-sync Unknown column: row(appcode varchar, appdataparams row(appcode varchar, environment varchar), backend row(hooks row(post array(row(calldata varchar, gaslimit varchar, target varchar)), pre array(row(calldata varchar, gaslimit varchar, target varchar)))), environment varchar, fullappdata varchar, metadata row(environment varchar, hooks row(post array(row(calldata varchar, gaslimit varchar, target varchar)), pre array(row(calldata varchar, gaslimit varchar, target varchar)), version varchar), orderclass row(orderclass varchar, version varchar), partnerfee row(bps bigint, recipient varchar), quote row(buyamount varchar, sellamount varchar, slippagebips varchar, version varchar), referrer row(address varchar, kind varchar, referrer varchar, version varchar), utm row(utmcampaign varchar, utmcontent varchar, utmmedium varchar, utmsource varchar, utmterm varchar, version varchar), widget row(appcode varchar, environment varchar)), metadataparams row(orderclassparams row(orderclass varchar), quoteparams row(slippagebips varchar), utmparams row(utmsource varchar)), version varchar) - treating as JSONB
2024-11-23 11:14:12,734 INFO dune-sync Data saved to app_data successfully!
2024-11-23 11:14:12,734 INFO dune-sync Job completed: Table Independent Query to Dune
2024-11-23 11:14:12,735 WARNING dune-sync Unknown column: array(row(address varbinary, storagekeys array(varbinary))) - treating as JSONB
2024-11-23 11:14:12,780 INFO dune-sync Data saved to results_4238114 successfully!
2024-11-23 11:14:12,780 INFO dune-sync Job completed: Sync Parameterized Dune Query with Multiple Types to Postgres
```